README.md

```diff
@@ -19,50 +19,63 @@ And then execute:
 ```ruby
 # spec_helper.rb
 RSpec.configure do |config|
-  # Include RSpec::Trailblazer::Matchers for operation tests
-  config.include RSpec::Trailblazer::Matchers, type: :operation
+  # Include RSpec::Trailblazer::Operation::Matchers for operation tests
+  config.include RSpec::Trailblazer::Operation::Matchers, type: :operation
 end
 ```
 
 ## Examples
 
 ```ruby
 class Post < ActiveRecord::Base
-  class Index < Trailblazer::Operation
-    include Collection
-    include CRUD
-    model Post
+  class Create < Trailblazer::Operation
+    extend Contract::DSL
```

Comment:
Sorry also this is deprecated - can you please use a "normal" contract instead to use this?

Reply:
Sure, I'll update that. I liked this way when we've only few properties in form.

Reply:
@yogeshjain999 I do too hahah

the problem is to _resuse_ the form, need to `Thing::Contract::Create['contract.default.class']`, weird no? hahahah

Reply:
Yeah right, haha

Reply:
I've updated it @emaglio , thanks for pointing it out.